com.transmissionbt.Transmission.appdata.xml

```diff
@@ -153,6 +153,7 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release date="2020-05-03" version="3.00" />
```

Comment:
The appdata was accepted upstream (though doesn't contain release dates etc). Perhaps rather than ignoring that, we could rename and patch it (and send the patches)?

Reply:
Done. I didn't add detailed change log as the list for 3.00 seems massive.